src/components/MediaSettings/MediaSettings.vue

```diff
@@ -310,13 +306,15 @@ export default {
 			stopDevices,
 			virtualBackground,
 			model: localMediaModel,
+			tabs,
+			isVirtualBackgroundAvailable,
```

Comment:
we don't need it outside of setup, right?

Reply:
No, can be removed

Reply:
If it was about `isVirtualBackgroundAvailable`

Reply:
Fixed